src/utils/wallet.js

```diff
@@ -205,21 +320,34 @@ class Wallet {
             meta: { type }
         }) => permission === 'FullAccess' && type !== 'ledger' && !(localAccessKey && public_key === localAccessKey.public_key))
 
+        const WALLET_METADATA_METHOD = '__wallet__metadata'
```

Comment:
why not have this inside of `this.addWalletMetadataAccessKeyIfNeeded`?

Reply:
I didn't really understand this comment or the pattern here. If you think it needs to be done, I'll need more context.

Reply:
```
 let newLocalKeyPair = await this.addWalletMetadataAccessKeyIfNeeded(this.accountId, localAccessKey)
```

This shows up as removed and the stuff that is inside of method is added back here. Is this intentional? Why not call `this.addWalletMetadataAccessKeyIfNeeded(this.accountId, localAccessKey)` instead?